addon/helpers/name-initials.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+const { Helper } = Ember;
+
+export function nameInitials(params) {
+  let [name] =  params;
+  let text = name.trim().toLowerCase();
+  return text.replace(/\s[^a-z]/gi, '').split(' ').reduce((acc, word) => {
+    let trademarkSign = acc.length > 0 && acc[acc.length - 1].startsWith('t')
+      && word.startsWith('m');
```

Comment:
Which case this is handle ? If the name is `t*** m***` this will be fall in the trademarkSign.  And instead of doing `acc[acc.length - 1].startsWith('t')` you can just do `acc[acc.length - 1] === 't'`